apps/mobile/src/libs/components/safe-area-screen-wrapper/safe-area-screen-wrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {
+	SafeAreaView,
+	useSafeAreaInsets,
+} from "react-native-safe-area-context";
+
+type Props = {
+	children: React.ReactNode;
+};
+
+const SafeAreaScreenWrapper: React.FC<Props> = ({ children }) => {
```

Comment:
And rename the file please
```suggestion
const ScreenWrapper: React.FC<Props> = ({ children }) => {
```